Tests/kaas/plugin/plugin_kind.py

```diff
@@ -21,7 +21,7 @@ def _create_cluster(self):
         if self.config is None:
             self.cluster.create()
         else:
-            self.cluster.create(self.config)
+            self.cluster.create()
```

Comment:
Why this change? It has nothing to do with cluster stacks.

Reply:
This change is currently outdated, as the code in #753 has also been modified.